research/conditional/utils/conditional_trainer.py

```diff
@@ -179,7 +204,35 @@ def train(self, n_steps: int):
                     except:
                         print("Decoding failed, skipping...")
                 self._after_step_operations(step)
-        self._after_train_operations()
+                if self.scheduler_trapezoidal_slides:
+                    for e in self.scheduler_trapezoidal_slides:
+                        if step == e["split_step"]:
+                            # dev delete splited, one of neptune loggers
+                            # dev save model and add to checkpoint
+                            splitted_loggers = None
```

Comment:
past participle of split is split (should probably be `split_loggers`).

Reply:
divert from config argument that defines step on which training is meant to be split

Reply:
done